pkg/registry/apis/provisioning/jobs/types.go

```diff
@@ -22,6 +22,9 @@ type JobQueue interface {
 	// Get the next job we should process
 	Next(ctx context.Context) *provisioning.Job
 
+	// Get the status for a job
+	Status(ctx context.Context, namespace string, name string) *provisioning.JobStatus
```

Comment:
^^ the controller loop now makes sure the correct status is written for running jobs.  Previously working jobs would stay "working" forever

Reply:
yes -- try something that takes longer to run :)